locale/en/site.json

```diff
@@ -72,16 +84,18 @@
             "link": "docs/faq",
             "text": "FAQ"
         },
-        "api": {
-            "text": "API"
+        "api-lts": {
+            "link": "dist/latest-v4.x/docs/api",
```

Comment:
Good catch @abouthiroppy. I accidentally introduced this while trying to figure out how to handle the "LTS" label on the version 4 docs. I'll push a patch shortly.